app/components/list-widget/component.js

```diff
@@ -188,7 +188,7 @@ export default Ember.Component.extend({
             }
```

Comment:
The ` transitionToFacet` action getting called is not this one, but the action on the `widget-adapter` which does take two parameters: https://github.com/cos-labs/share-analytics/blob/master/app/components/widget-adapter/component.js#L563